docs/compression_algorithms/Quantization.md

```diff
@@ -109,6 +109,17 @@ The quantization configuration in the `"target_device": "NONE"` case may be over
 
 For all target HW types, parts of the model graph can be marked as non-quantizable by using the `"ignored_scopes"` field - inputs and weights of matching nodes in the NNCF internal graph representation will not be quantized, and the downstream quantizers will not propagate upwards through such nodes.
 
+---
+**NOTE**
+
+There is a known issue with AVX2 and AVX512 CPU devices. The issue appears with 8-bit matrix calculations with tensors which elements are close to the maximum or saturated.
+AVX2 and AVX512 utilize a 16-bit register to store the result of operations on tensors. In case when tensors are saturated the buffer overflow happens.
+This leads to accuracy degradation.
+
+To fix this issue inside NNCF, weight tensors are quantized in 8 bits but only 7 bits are effectively used. 
```

Comment:
We don't have `target_device: "CPU_VNNI"` in the OpenVINO post-training tool. For alignment, I suggest not introducing it. You have any concerns about having this option, please explain, i.e. what performance/accuracy benefits we can have with it.

Reply:
To align with OpenVINO post-training tool I removed `target_device: "CPU_VNNI"`. But we might keep in mind that this can be useful in the future